src/gui/guiFormSpecMenu.cpp

```diff
@@ -3629,6 +3685,14 @@ bool GUIFormSpecMenu::OnEvent(const SEvent& event)
 						return true;
 					}
 
+					if (!s.url.empty()) {
+						if (m_client) {
+							errorstream << "Unable to use button_url in game!" << std::endl;
```

Comment:
it sounds a little bit hacky to have those m_client tests

Reply:
Is there a better way to detect if we're in the menu or not?